neo/Consensus/ConsensusService.cs

```diff
@@ -281,9 +285,10 @@ private void OnRecoveryMessageReceived(ConsensusPayload payload, RecoveryMessage
         {
             if (message.ViewNumber < context.ViewNumber) return;
             Log($"{nameof(OnRecoveryMessageReceived)}: height={payload.BlockIndex} view={message.ViewNumber} index={payload.ValidatorIndex}");
+            if (context.WatchOnly()) return;
```

Comment:
OnRecoveryMessageReceived should occur naturally, we need to remove this exit `if (context.WatchOnly()) return;`

Reply:
If a node is watch-only, it can wait for the next block for recovery.

Reply:
I agree, but I think it will will lose some part of the history for the creation of the block.

Reply:
Done.